addons/hr_skills/__manifest__.py

```diff
@@ -36,11 +36,12 @@
         'data/hr.resume.line.csv',
     ],
     'installable': True,
+    'auto_install': True,
```

Comment:
Ho why?

Reply:
That's not an answer. Why having a separated module, then?

Reply:
Hi, i took back the task and you're right, modules should be merged actually if we enable it by default, i'll discuss this with the PO as to either disable auto install or merge modules

Reply:
Hey Hey @KangOl, sorry for that, that is not an answer indeed :woman_facepalming: 

1) It should be separate module, because it's conceptually something else then employee. Also, in case user really doesn't need it, they have the option to remove it. Imagine being a real small company, you might not care much for skills.

2) Now, why autoinstall? Because in saas, a _typical_ user does usually checks settings. So, what happens for a typical user? They see employee profile without results and skills, and they have a feeling that something is missing. (Also for demo data..to demonstrate this feature well)

 It's a better to give a good first impression of a full product, while also giving an opportunity to remove it, if needed.

What do you think? :thinking: